src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -10,12 +10,41 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import { _ } from 'okta';
+
 /**
  * Example of the option like
  * @param {AuthenticatorOption[]} options
  * @param {( AuthenticatorEnrollment[] || Authenticator[] )} authenticators
  */
+const createOVOptions = (options = []) => {
+  // Split OV into individual entries for verification (one for each method).
+  const ovItem = options.find((option) => option.relatesTo.type === 'app');
+  const methodTypeObj = ovItem?.value?.form?.value?.find(v => v.name === 'methodType');
+  const methodOptions = methodTypeObj?.options;
+  if (methodOptions) {
+    const ovOptions = [];
+    methodOptions.map((method) => {
```

Comment:
- probably can just be `const ovOptions = methodOptions.map(() => { ...; return Object.assign({...}); })`
- Add comments to explain what/why this method does